### PR TITLE
opensnitch-ui: fix events table and notifications

### PR DESCRIPTION
**Pull request by RamKromberg**
###### Description of changes

The events table (first tab) in opensnitch-ui stopped populating in 22.05 and allow/block notification failed to pop, leaving users scratching their heads wondering why the internet isn't working :D

The issue was traced back to bad imports. Upstream had this sorted in the Makefile: https://github.com/evilsocket/opensnitch/blob/master/ui/Makefile#L8
So I've done the same: https://github.com/RamKromberg/nixpkgs/blob/a82364af80ac25d4f0d7ef672203b760d582d987/pkgs/tools/networking/opensnitch/ui.nix#L41

Additionally, the IPASN data files were missing from the pyasn's pypy package so lookups weren't being performed. Arch copied them from the github package so I followed their example: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=python-pyasn

Otherwise, I also tweaked the site-package strings so they won't break over future version changes.

NOTE: Please backport to stable as opensnitch-ui is currently broken.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/opensnitch/ui.nix
Comment (onny) [inline pkgs/tools/networking/opensnitch/ui.nix:41]: Maybe adding a comment here that this is already fixed upstream and not need in versions later 1.5.1

**Review by onny — COMMENTED** [review]
Thank you for fixing and further enhancing these packages :) Works great for me

File: pkgs/tools/networking/opensnitch/ui.nix
Comment (RamKromberg) [reply pkgs/tools/networking/opensnitch/ui.nix:41]: Oh it's been there for a while: https://github.com/evilsocket/opensnitch/blob/1.4.0/ui/Makefile

It's just that it never caused noticeable breakage until now.

I'll add a note referencing the Makefile. But, beyond that, calling the Makefile directly instead of doing it in the .nix involves more dependencies (for the i18 translations build) so I'll leave that to the permanent maintainers.

**Review by onny — APPROVED** [review]

**Comment by raboof** [comment]
> Additionally, the IPASN data files were missing from the pyasn's pypy package so lookups weren't being performed. Arch copied them from the github package so I followed their example: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=python-pyasn

Hmm. If I'm reading https://github.com/hadiasghari/pyasn#ipasn-data-files correctly these are test resources and sample databases, and it seems they intentionally didn't include them in the python package. On the other hand, opensnitch indeed expects them to be there (https://github.com/evilsocket/opensnitch/commit/42b8774f52c62ce03cdc4463677038b075e7dc8b). I think we should raise this issue upstream (or perhaps the Arch maintainer already did that? I couldn't find it).

Rather than putting it in the `pyasn` package, perhaps we should instead introduce a separate `pyasn-sample-data` package or something like that?

**Review by raboof — APPROVED** [review]
Great to have the imports fix - sad that we didn't notice that.

I think the pyasn workaround could use some improvements, but since this PR fixes an important problem and @onny, pyasn's maintainer, already approved this PR, I think it's OK to merge this and make any improvements as follow-up PR's.

**Comment by raboof** [comment]
Result of `nixpkgs-review pr 175954` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>opensnitch-ui</li>
    <li>python310Packages.pyasn</li>
    <li>python39Packages.pyasn</li>
  </ul>
</details>

**Comment by RamKromberg** [comment]
thx. @raboof stable is currently broken too so backporting would also be welcomed :)

**Pull request merged**
